File: liqp-core/src/main/java/liqp/tags/Comment.java
package liqp.tags;

import liqp.tag.LTag;
import liqp.node.LNode;
import liqp.context.LContext;

public class Comment extends LTag {

    /*
     * Block tag, comments out the text in the block
     */
    @Override
    public Object render(LContext context, LNode... nodes) {
        return "";
    }
}

File: liqp-core/src/main/kotlin/liqp/tags/Comment.kt
package liqp.tags

import liqp.ControlResult
import liqp.context.LContext
import liqp.node.LNode
import liqp.tag.LTag

class Comment : LTag() {

  /**
   * Block tag, comments out the text in the block
   */
  override fun render(context: LContext, vararg nodes: LNode): Any? {
    return ControlResult.NO_CONTENT
  }
}
